### ISSUE TITLE
Does not work b_client_skip_channelgroup_permissions

### ISSUE THREAD

**Issue by REDMONDS** [TeaSpeak/TeaSpeak#415]
**Describe the bug**
When creating the channel and installing `i_channel_needed_join_power=150`, the Admin server can not enter the channel.

**To Reproduce**
Steps to reproduce the behaviour:
1. We go to the server and create a new channel.
2. Install `i_channel_needed_join_power=150` and click Apply.
3. We go to the previously created channel
4. see the error in the screenshot.

**Expected behaviour**
The grafting `b_client_skip_channelgroup_permissions` should work and we go to the channel.

**Screenshots**
1. https://prnt.sc/ksb836
2. https://prnt.sc/ksb8ae

**Environment information:**
 Operating system: Debian 9.4
 TeaSpeak version: TeaSpeak-1.2.16-beta-1
 I am using `experimental_31`: yes

**Additional context**
Reproduced on a clean server, that is, after it is raised.

**Comment by WolverinDEV** [comment]
Well this is again a non topical acting from TeamSpeaks permission system.
Is there any need to implement it like that?

**Comment by REDMONDS** [comment]
@WolverinDEV Yes, there is a need for this. The main administrators should ignore the rights of the channels and go into them without problems.
I have a lot of channels on the server, which users have configured and information about non-working channels came from them.

**Comment by WolverinDEV** [comment]
Hmm I think a workaround for an invalid configuration specially like this exception is a little it stupid.
A new permission like 'i_client_channel_ignore_join_power' would make more sense.
I'll think and discuss about that, but I'll definitely not adjust the system like TeamSpeak because they made there ab exception which is more or less like onion fixing :)

**Comment by REDMONDS** [comment]
`b_client_skip_channelgroup_permissions` - Skip Channel Group Permissions

I would like to of course restore this privilege, let's hope.
![default](https://user-images.githubusercontent.com/6460267/45278364-67f5dd00-b4d5-11e8-8364-3e051d4c78ae.png)
